### PR TITLE
Gradle

### PR DESCRIPTION
**Pull request by SgtSilvio**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,62 +1,67 @@
...
 
-Please visit {hivemq-extensions-docu-link}[our website] for the documentation how to develop extensions.
+Please visit the extensive {hivemq-extensions-docu}[HiveMQ extension developer documentation].
+It covers the core concepts and shows how to set up your development environment.
 
-=== Example Extensions
+== Example Extensions
 
-Many example extensions are available, most of them as open source. You can find an overview of officially supported extensions {hivemq-extensions-directory}[here]:
+You can find many example extensions by browsing the {hivemq-github}[HiveMQ Github page].
```
Comment (sauroter) [inline README.adoc:52]: Is this not the _Github space_?

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+dependencies {
+    compile group: 'io.dropwizard.metrics', name: 'metrics-core', version: '4.0.2'
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
+}
+
+
+/* ******************** java ******************** */
+
+sourceCompatibility = JavaVersion.VERSION_1_10
```
Comment (sauroter) [inline build.gradle:45]: 11?

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+                        organizationUrl = project.url
+                    }
+                    developer {
+                        id = 'lbrandl'
+                        name = 'Lukas Brandl'
+                        email = 'lukas.brandl@hivemq.com'
+                        organization = project.vendor
+                        organizationUrl = project.url
+                    }
+                    developer {
```
Comment (sauroter) [inline build.gradle:128]: add Silvio Giebl and Georg Held. I added them in the CE as well.

File: src/main/java/com/hivemq/extension/sdk/api/ExtensionMain.java [hunk 4]
```diff
@@ -28,53 +28,58 @@
...
      * It can be used to setup the extension and register client interceptors and callbacks with HiveMQ.
      * <p>
      * If this method returns HiveMQ assumes that this extension is finished starting up.
      * <p>
-     * There are multiple options when this method can be called by HiveMQ:<br/>
+     * There are multiple options when this method can be called by HiveMQ:
      * <ul>
-     * <li>When HiveMQ starts up</li>
-     * <li>When the extension is enabled at runtime due to an administrative action</li>
+     *   <li>When HiveMQ starts up
```
Comment (sauroter) [inline src/main/java/com/hivemq/extension/sdk/api/ExtensionMain.java:49]: Why is it removing the closing tags?

Comment (SgtSilvio) [reply src/main/java/com/hivemq/extension/sdk/api/ExtensionMain.java:49]: Javadoc is all about readability (even in the source code)
I read about javadoc coding standards, for example: https://blog.joda.org/2012/11/javadoc-coding-standards.html

File: build.gradle (hunk 2 above)
Comment (SgtSilvio) [reply build.gradle:45]: Yes, it was 10 a few weeks ago

File: README.adoc (hunk 1 above)
Comment (SgtSilvio) [reply README.adoc:52]: I don't know if the term GitHub space exists, but will change it.

**Pull request merged**